confidence/models.py

```diff
@@ -46,11 +48,11 @@ def __init__(self, *sources, separator='.', missing=Missing.silent):
         self._missing = missing
         self._root = self
 
-        if isinstance(self._missing, Missing):
+        if isinstance(missing, Missing):
```

Comment:
Maybe now the type of `self._missing` is `Any`, this can be changed back to `self._missing`; seems odd to check the parameter instead of the member here (though it makes no functional difference).

Reply:
FXIED